### DISCUSSION TITLE
JXR support

### DISCUSSION THREAD

**Discussion by LMBits** [ccseer/Seer#384] [General]
Any chance this could be added? Nvidia HDR screenshots uses this format. Windows 11 explorer preview them just fine.

**Comment by ccseer** [comment]
might work on it for the Scripts module, as an example

**Reply by LMBits** [reply]
I am sorry I am not familiarized with the Scripts module. What is it? I only know of the plugins section. This is what I found in ImageMagick website. Maybe that would be an easy solution. Would you mind sharing how that would be done?

JXR | RW | JPEG extended range | Requires the jxrlib delegate library. Put the JxrDecApp and JxrEncApp applications in your execution path.

**Reply by ccseer** [reply]
https://github.com/ccseer/Seer/wiki/7.-Scripts

**Reply by ccseer** [reply]
you can try it here https://github.com/ccseer/Seer/discussions/382

**Reply by LMBits** [reply]
That takes you to a 3.1.2 version. Couldn't find any scripts section in the settings.

**Reply by ccseer** [reply]
![image](https://user-images.githubusercontent.com/15963166/210294737-5c0c1922-6712-404d-a2e8-5f96ff9355b4.png)

**Reply by LMBits** [reply]
I don't see that in the version I just installed from the link you mentioned. It doesn't say to be 3.2.0.

**Reply by ccseer** [reply]
You need to quit Seer then install again. The installer can't kill the running exe on Win10 now
![image](https://user-images.githubusercontent.com/15963166/210295172-86e2e904-2058-4cb1-a4a4-6b99c410fe7e.png)

**Reply by LMBits** [reply]
It seems that was the problem. Thanks.

**Comment by LMBits** [comment]
I have just tried adding the epub script and it works perfectly fine. Looking forward to hearing back if you get to make the script for the format suggested. Thanks a lot for explaining how that works and looking into it.

**Reply by ccseer** [reply]
Please let me know if you found any bug. It's a beta version.

I still need a few days to prepare two or three more scripts.

**Reply by LMBits** [reply]
Sure I will. Thanks again.

**Comment by ccseer** [comment]
added in 4.0.0

![image](https://github.com/user-attachments/assets/110e8ac5-0abd-488a-81c3-345fbbcee2a1)
